### ISSUE TITLE
Add "Copy link" to "Login required" dialog

### ISSUE THREAD

**Issue by baznikin** [DeedleFake/trayscale#159]
Use cases:
1. I am testing new headscale/tailscale installation and I'm logging in into my OIDC provider with various users. It's simpler for me to open auth link in private mode instead of logging out as admin user
2. Browser do not opens for me anyway. I do not see any error messages in logs - neither app nor system. It's better to have possibility to get connection URL for such situations

**Comment by DeedleFake** [comment]
Good idea. I'll see about adding it for people to copy manually if necessary.
